### DISCUSSION TITLE
Bar charts for Insights

### DISCUSSION THREAD

**Discussion by diegogurpegui** [directus/directus#16123] [Feature Requests]
Insights is a very good tool for quick and simple reporting on the data, and it could be great with a few more chart options.
I think "Bar charts" is the next step. The way I see it the user would need to:

- Select the collection (obviously)
- Select the series field, and the aggregate function
- Select the grouping field (what each bar will represent)
  The grouping can be more complex, but just selecting a field could be a good start.

Also the possibility to set filters, as with everything else, should be there too.
The bars could be limited to always vertical, but if it's simple additional configurations can be included.

Note: I'm sure this is an obvious idea and many thought of it but I couldn't find any reference to this. If this is already in the works or if there's a reason for this to not being there, please let me know and close this :)

**Comment by benhaynes** [comment]
Love it! This is actually in progress now...

**Reply by timchambers** [reply]
This is great news, any sense of when additional chart types will be available?

**Comment by rijkvanzanten** [comment]
This is launching tomorrow!
